.terserrc

```diff
@@ -0,0 +1,7 @@
+{
+	"mangle": false,
+	"format": {
+		"beautify": true,
```

Comment:
I'm not sure why this isn't being followed, I think it worked in previous Parcel betas. Not fundamental though.

- Docs: https://parceljs.org/languages/javascript/#minification
- Previously used in https://github.com/fregante/browser-extension-template/pull/44